.github/workflows/release.yml

```diff
@@ -1,26 +1,33 @@
 name: Release
 
 on:
-  release:
-    types: [published]
+  push:
+    branches:
+      - main
 
 concurrency: ${{ github.workflow }}-${{ github.ref }}
 
 jobs:
   release:
     name: Release
+
+    # IMPORTANT: prevent this action from running on forks
+    if: ${{ github.repository_owner == 'formspree' }}
+
     runs-on: ubuntu-latest
     steps:
       - name: Checkout Repo
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
-      - name: Setup Node.js 16.x
-        uses: actions/setup-node@v2
+      - name: Setup Node.js 18.x
+        uses: actions/setup-node@v3
         with:
-          node-version: 16.x
```

Comment:
16 is in maintenance. Let's use the current LTS. https://github.com/nodejs/release#release-schedule